### PR TITLE
update: clean up main branch

### PR DESCRIPTION
**Pull request by patrickzheng200**
1. Deprecated the old verification workflow.
2. Removed usage of cache.

Signed-off-by: Patrick Zheng <patrickzheng@microsoft.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,10 +1,10 @@
 module github.com/notaryproject/notation-go
 
-go 1.18
+go 1.19
```
Comment (shizhMSFT) [inline go.mod:3]: The golang version should not be bumped up.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by shizhMSFT — CHANGES_REQUESTED** [review]
Let's revert the go version bump up in the `go.mod` before merging.

**Review by rgnote — APPROVED** [review]

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**